### ISSUE TITLE
A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader:NVDA does not announce the tittle of the screen as soon as it is landed on the screen.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7673]
**GitHubTags:** #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#DesktopApp;#Win32;#NVDA;
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Repro Steps:**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster”.
5. "Select the deployment option" screen gets open.
6. Start NVDA and navigate with the help of up and down arrow and observe the announcement.
**Actual:**
NVDA does not announce the tittle of the screen as soon as it is landed on the screen.
**Expected:**
NVDA should announce the new screen name as soon as it is landed on the new screen.
**User Impact:**
Screen reader User will not understand weather they have landed on new screen or not. 

**Attachment for Reference:**
[7673_A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3717400/7673_A11y_AzureDataStudio_Selectthedeploymentoption_ScreenReader.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by V-sheman** [comment]
Verified the bug issue is fixed does not repro.
![image](https://user-images.githubusercontent.com/56344899/72047464-927dc300-32e0-11ea-8b6e-dbe87abb5137.png)

**Comment by V-sheman** [comment]
Verified the bug Version: 1.15.0-insider (user setup) issue is fixed hence closing the bug from tester end. 
![image](https://user-images.githubusercontent.com/56344899/72047933-a37b0400-32e1-11ea-896e-c58a957f73d8.png)
